tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -491,7 +508,7 @@ def test_aliased_fields(self):
         assert data[0]["count_id"] == 1
         assert data[0]["count_unique_user"] == 1
         assert "latest_event" in data[0]
-        assert "project.name" in data[0]
+        assert "project.name" not in data[0]
```

Comment:
Not sure why we returned `project.name` when it wasn't a requested field. I might be wrong in removing it if its required to be returned regardless, but I can't tell if that was intentional or not.

Reply:
We need project.name to generate links to event details, as events are only unique within a project.

Reply:
:+1: Missed the `auto_field` parameter. `project.name` will now get added automatically when its True.